### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by KeilChris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,10 @@
-# CMSIS Toolbox Installer
+# CMSIS-Toolbox Installer
 
-This is the installer for the CMSIS-Toolbox.
+This is the installer for the CMSIS-Toolbox. It provides command line tools for processing software packs that are provided in Open-CMSIS-Pack format. It is currently under development but supports already a wide range of use cases (refer to [**implementation progress**](./docs/progress.md)).
```
Comment (jkrech) [inline README.md:3]: The tools do not just process software packs but it is also to process projects and configure build systems to create target binaries.

**Review by jkrech — CHANGES_REQUESTED** [review]
Could you please respond to my comment.

File: README.md (hunk 1 above)
Comment (KeilChris) [reply README.md]: Updated REDME.md with more details.

**Review by jkrech — APPROVED** [review]
Thanks. This looks clearer now.

**Pull request merged**